docs/subnets/how-to-list-and-describe.md

```diff
@@ -0,0 +1,170 @@
+# How to View Your Created Subnets
+
+## List Subnet Configurations
+
+You can list the Subnets you've created with
+
+`avalanche subnet list`
+
+Example:
+
+```text
+> avalanche subnet list
++----------+-----------+------------+-----------+-----------+-------+----------+---------+
+|  SUBNET  |   CHAIN   |  CHAIN ID  |   TYPE    | FROM REPO |       | DEPLOYED |         |
++----------+-----------+------------+-----------+-----------+-------+----------+---------+
+|          |           |            |           |           | Local | Fuji     | Mainnet |
++----------+-----------+------------+-----------+-----------+-------+----------+---------+
+| mySubnet | mySubnet  |       1000 | SubnetEVM | false     | No    | No       | No      |
++----------+-----------+------------+-----------+-----------+-------+----------+---------+
+```
+
```

Comment:
do we need to explain the columns? probably yes

Reply:
For some reason I don't think this is the right place to do this. I think we might need a subnet glossary. I'll create a ticket.

Reply:
https://github.com/ava-labs/avalanche-cli/issues/481